dev/fattest.simplicity/src/componenttest/topology/utils/tck/TCKResultsWriter.java

```diff
@@ -36,7 +36,7 @@ public class TCKResultsWriter {
      *
      * @param resultInfo TCK Results metadata
      */
-    public static void preparePublicationFile(TCKResultsInfo resultInfo) {
+    public static void preparePublicationFile(TCKResultsInfo resultInfo, String repeat) {
```

Comment:
please update the javadoc too

Reply:
be clear if a null repeat string is allowed/expected

Reply:
instead of adding the repeat param here, it might be better to include the repeat string as part of `TCKResultsInfo`

Reply:
Moved to `TCKResultsInfo`.

A `null` will never be returned from the [RepeatTestFilter.getRepeatActionsAsString() ](https://github.com/OpenLiberty/open-liberty/blob/d1f0e4e8064d91deef53dfc06700ea667d0bd4ee/dev/fattest.simplicity/src/componenttest/custom/junit/runner/RepeatTestFilter.java#L151) because if the `REPEAT_ACTION_STACK` is empty the string is set to be empty.